code/modules/awaymissions/corpse.dm

```diff
@@ -248,7 +248,44 @@
 
 
 
-///////////Civilians//////////////////////
+/////////////////Crew//////////////////////
+
+/obj/effect/landmark/corpse/assistant
+	name = "Assistant"
+	corpseuniform = /obj/item/clothing/under/color/grey
+	corpseshoes = /obj/item/clothing/shoes/black
+	corpsebelt = /obj/item/weapon/storage/bag/plasticbag
+	corpseback = /obj/item/weapon/storage/backpack
+	corpseradio = /obj/item/device/radio/headset
+	corpseid = 1
+	corpseidjob = "Assistant"
+	corpseidaccess = "Assistant"
+
+/obj/effect/landmark/corpse/assistant/robust
+	corpseuniform = list(/obj/item/clothing/under/syndicate, /obj/item/clothing/under/rank/captain, /obj/item/clothing/under/rank/head_of_personnel, /obj/item/clothing/under/rank/head_of_security, /obj/item/clothing/under/rank/chief_medical_officer, /obj/item/clothing/under/rank/chief_engineer)
+	corpsesuit = list(/obj/item/clothing/suit/space/ancient, /obj/item/clothing/suit/space/rig/ror, /obj/item/clothing/suit/space/rig/syndi, /obj/item/clothing/suit/space/rig/atmos/, /obj/item/clothing/suit/armor/captain, /obj/item/clothing/suit/armor/vest, /obj/item/clothing/suit/armor/riot, /obj/item/clothing/suit/armor/laserproof, /obj/item/clothing/suit/armor/hos, /obj/item/clothing/suit/fire/firefighter, /obj/item/clothing/suit/space/rig/security, /obj/item/clothing/suit/space/rig/mining)
+	corpseshoes = list(/obj/item/clothing/shoes/magboots, /obj/item/clothing/shoes/magboots/atmos, /obj/item/clothing/shoes/magboots/captain, /obj/item/clothing/shoes/jackboots/knifeholster, /obj/item/clothing/shoes/galoshes)
+	corpsegloves = list(/obj/item/clothing/gloves/yellow/power, /obj/item/clothing/gloves/yellow/power, /obj/item/clothing/gloves/yellow/power, /obj/item/clothing/gloves/yellow/power, /obj/item/clothing/gloves/black, /obj/item/clothing/gloves/captain)
```

Comment:
Yeah these aren't insulateds these are the emperor palpatine tator engineer gloves